server/server.go

```diff
@@ -768,29 +768,39 @@ func (s *Server) JoinPeer(rawPeerMultiaddr string) error {
 
 // Close closes the Minimal server (blockchain, networking, consensus)
 func (s *Server) Close() {
```

Comment:
Make some notes why we should change close sequence?

Reply:
ok, added it [272b4cd](https://github.com/dogechain-lab/dogechain/pull/256/commits/272b4cd28d599471011a25535c593e5950073f70)